frontends/web/src/locales/en/app.json

```diff
@@ -197,6 +197,40 @@
   "bitboxBase": {
     "connect": "Connect Base",
     "detectedBases": "Detected the following devices on your network:",
+    "errors": {
+      "CHANGEPASSWORD_TOO_SHORT": "Password too short. A password must be at least 8 characters.",
+      "EXECUTABLE_NOT_FOUND": "Internal error: EXECUTABLE_NOT_FOUND",
```

Comment:
Thanks!
I have one suggestion. For the internal errors, let's do this:
`"EXECUTABLE_NOT_FOUND": "Internal error: {{error}}",`
so we can actually pass in the error code. Don't want the translators to accidentally translate the codes you put in there :)

You can also run `make locize-fix` to fix the formatting (mostly looks like missing spaces after colons) and then fixup and force push :)